tests/Cookie/ParseTest.php

```diff
@@ -143,96 +243,145 @@ public function testParsingHeaderObject($header, $expected, $expected_attributes
+				'expected_attributes' => ['expires' => gmmktime(4, 50, 12, 12, 5, 2013)],
+			],
+			'Expiration date parsing: format: RFC 850, obsoleted by RFC 1036; non-expired date' => [
+				'header'              => 'foo=bar; Expires=Friday, 5-Dec-2014 04:50:12 GMT',
+				'expected'            => ['expired' => false],
+				'expected_attributes' => ['expires' => gmmktime(4, 50, 12, 12, 5, 2014)],
+			],
+			// Test with asctime().
+			'Expiration date parsing: format: asctime(); expired date' => [
+				'header'              => 'foo=bar; Expires=Thu Dec  5 04:50:12 2013',
+				'expected'            => ['expired' => true],
+				'expected_attributes' => ['expires' => gmmktime(4, 50, 12, 12, 5, 2013)],
+			],
+			'Expiration date parsing: format: asctime(); non-expired date' => [
+				'header'              => 'foo=bar; Expires=Fri Dec  5 04:50:12 2014',
+				'expected'            => ['expired' => false],
+				'expected_attributes' => ['expires' => gmmktime(4, 50, 12, 12, 5, 2014)],
+			],
+			// Invalid.
+			'Expiration date parsing: invalid expiration date' => [
+				'header'              => 'foo=bar; Expires=never',
+				'expected'            => [],
+				'expected_attributes' => ['expires' => null],
+			],
+
+			/*
+			 * Max-Age parsing.
+			 */
+			'Max-Age parsing: value: 10' => [
+				'header'              => 'foo=bar; Max-Age=10',
+				'expected'            => ['expired' => false],
+				'expected_attributes' => ['max-age' => gmmktime(0, 0, 10, 1, 1, 2014)],
+			],
+			'Max-Age parsing: value: 3660' => [
+				'header'              => 'foo=bar; Max-Age=3660',
+				'expected'            => ['expired' => false],
+				'expected_attributes' => ['max-age' => gmmktime(1, 1, 0, 1, 1, 2014)],
+			],
+			'Max-Age parsing: value: 0' => [
+				'header'              => 'foo=bar; Max-Age=0',
+				'expected'            => ['expired' => true],
+				'expected_attributes' => ['max-age' => 0],
+			],
+			'Max-Age parsing: value: 1000' => [
+				'header'              => 'foo=bar; Max-Age=-1000',
+				'expected'            => ['expired' => true],
+				'expected_attributes' => ['max-age' => 0],
+			],
+			// Invalid (non-digit character).
+			'Max-Age parsing: value: 1e6 (non-digit)' => [
+				'header'              => 'foo=bar; Max-Age=1e6',
+				'expected'            => ['expired' => false],
+				'expected_attributes' => ['max-age' => null],
+			],
+
+			/**
+			 * Arbitrary attributes.
+			 */
+			'Attribute parsing where the attribute does not contain an = sign' => [
+				'header'              => 'foo=bar; HttpOnly',
+				'expected'            => [],
+				'expected_attributes' => ['httponly' => true],
+			],
+			'Attribute parsing: surrounding whitespace should be stripped' => [
+				'header'              => 'foo=bar; ArbitraryName   =  some-value   ',
+				'expected'            => [],
+				'expected_attributes' => ['ArbitraryName' => 'some-value'],
+			],
+
+			/**
+			 * Alltogether now: multiple attributes in one cookie header.
```

Comment:
Only one 'l' for altogether.
```suggestion
			 * Altogether now: multiple attributes in one cookie header.
```